packages/expo-av/android/src/main/java/expo/modules/av/player/PlayerData.java

```diff
@@ -208,6 +222,16 @@ public static PlayerData createUnloadedPlayerData(final AVManagerInterface avMod
     }
   }
 
+  @UiThread
+  private void emitSampleBufferEvent(final byte[] bytes, final double currentPositionSeconds) {
+    final UIManager uiManager = mUiManager.get();
+    if (uiManager != null) {
+      uiManager.runOnClientCodeQueueThread(() -> {
+        sampleBufferCallback(bytes, currentPositionSeconds);
+      });
+    }
```

Comment:
The `currentPositionSeconds` passed into an async call will no longer be accurate, hopefully this isn't a significant loss